pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuild.java

```diff
@@ -88,6 +88,10 @@ public void addBuildLog(String log) {
         buildLog = log == null ? Collections.emptyList() : asList(log.split("\\r?\\n"));
```

Comment:
is this method still used?

Reply:
No, ATM it isn't used. Can I remove it or you or do you prefer to keep it for future use?